### PR TITLE
Add use of rOpenSci forum to maintenance section

### PR DESCRIPTION
**Pull request by maelle**
cf #141

### CODE REVIEW

File: maintenance_collaboration.Rmd [hunk 1]
```diff
@@ -16,10 +16,12 @@ We require that you use a code of conduct such as the [Contributor Covenant](htt
...
+We have templates for issue, pull request and contributing guidelines that you can find [in this GitHub repository](https://github.com/ropenscilabs/rodev/tree/master/inst/templates) along with [a helper function to insert them into your repository](https://ropenscilabs.github.io/rodev/reference/use_ro_github.html).
 
 You can tweak them a bit depending on your workflow and package. For example, make sure contributors have instructions in CONTRIBUTING.md for running local tests if not trivial. CONTRIBUTING.md can also contain some details about how you acknowledge contributions (see [this section](#attributions)) and the roadmap of your package (cf [this example](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md)).
 
+We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/). Users can use the forum to ask questions about use, and report their use cases, and where you can subscribe to categories and manage your notifications. Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.
```
Comment (noamross) [inline maintenance_collaboration.Rmd]:
```suggestion
We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/). Users can use the forum to ask questions about use, and report their use cases, and where you can subscribe to categories and manage your notifications.  Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.  Please direct your users to tag posts with the package name.
```

**Comment by maelle** [comment]
Further comments @noamross?

@sckott you might also want to have a look given you maintain the forum :smile_cat:

File: maintenance_collaboration.Rmd [hunk 2]
```diff
@@ -16,10 +16,12 @@ We require that you use a code of conduct such as the [Contributor Covenant](htt
...
 
 You can tweak them a bit depending on your workflow and package. For example, make sure contributors have instructions in CONTRIBUTING.md for running local tests if not trivial. CONTRIBUTING.md can also contain some details about how you acknowledge contributions (see [this section](#attributions)) and the roadmap of your package (cf [this example](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md)).
 
+We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/). Users can use the forum to ask questions about use, and report their use cases, and where you can subscribe to categories and manage your notifications.  Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.  Please direct your users to tag posts with the package name.
```
Comment (noamross) [inline maintenance_collaboration.Rmd]:
```suggestion
We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/). Users can use the forum to ask questions about use and report their use cases, and you can subscribe to individual categories and tags to receive notifications about your package.  Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.  Please direct your users to tag posts with the package name.
```

**Comment by noamross** [comment]
Made a slight grammar edit. Other than that, LGTM.

**Comment by maelle** [comment]
Oops forgot to wait for Scott's review :scream_cat: Scott, I'll implement any change you suggest directly in the dev branch, sorry!!

**Comment by sckott** [comment]
no problem, meant to get to this, but family stuff came up. LGTM

**Pull request merged**